samples/app-anonymous-users/csharp/Readme.md

```diff
@@ -21,11 +21,6 @@ This sample shows anonymous users support in meeting apps.
 **Interaction with app**
 ![appanonymoususersGif](Images/anonymoususersupport.gif)
 
-## Try it yourself - experience the App in your Microsoft Teams client
```

Comment:
Let's revert it once deployment issue has resolved.

Reply:
Ok